src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -84,6 +84,14 @@ const getRelationParts = (val: unknown): string[] => {
 
 // Extract wds URL from Leo response. exported for testing
 export const getWdsUrl = apps => {
+  // TODO: AJ-761: Check for if app.status === 'PROVISIONING'
```

Comment:
I think this info will be used for a different ticket than 761 - probably https://broadworkbench.atlassian.net/browse/AJ-783

Reply:
Updated!